webui/react/src/components/UserSettings.tsx

```diff
@@ -235,18 +230,14 @@ const UserSettings: React.FC<Props> = ({ show, onClose }: Props) => {
                   </Select>
                 </InlineForm>
                 <InlineForm<RowHeight>
-                  initialValue={experimentListGlobalSettings.rowHeight}
+                  initialValue={dataGridGlobalSettings.rowHeight}
                   label="Table Density"
                   valueFormatter={(rh) => rowHeightLabels[rh]}
                   onSubmit={(rh) => {
-                    userSettings.set(
-                      experimentListGlobalSettingsConfig,
-                      experimentListGlobalSettingsPath,
-                      {
-                        ...experimentListGlobalSettings,
-                        rowHeight: rh,
-                      },
-                    );
+                    userSettings.set(dataGridGlobalSettingsConfig, dataGridGlobalSettingsPath, {
+                      ...dataGridGlobalSettings,
+                      rowHeight: rh,
+                    });
```

Comment:
can we use `setPartial` here?

Reply:
When I try I get the following errors:
```
No overload matches this call.
  Overload 1 of 2, '(type: TypeC<{ rowHeight: ValueOfC<{ readonly EXTRA_TALL: "EXTRA_TALL"; readonly MEDIUM: "MEDIUM"; readonly SHORT: "SHORT"; readonly TALL: "TALL"; }>; tableViewMode: UnionC<[LiteralC<"scroll">, LiteralC<...>]>; }>, key: string, value: TypeOfPartialProps<...>): void', gave the following error.
    Argument of type 'PartialC<{ rowHeight: ValueOfC<{ readonly EXTRA_TALL: "EXTRA_TALL"; readonly MEDIUM: "MEDIUM"; readonly SHORT: "SHORT"; readonly TALL: "TALL"; }>; tableViewMode: UnionC<[LiteralC<"scroll">, LiteralC<...>]>; }>' is not assignable to parameter of type 'TypeC<{ rowHeight: ValueOfC<{ readonly EXTRA_TALL: "EXTRA_TALL"; readonly MEDIUM: "MEDIUM"; readonly SHORT: "SHORT"; readonly TALL: "TALL"; }>; tableViewMode: UnionC<[LiteralC<"scroll">, LiteralC<...>]>; }>'.
      Types of property '_tag' are incompatible.
        Type '"PartialType"' is not assignable to type '"InterfaceType"'.
  Overload 2 of 2, '(type: IntersectionType<[TypeC<Props>, PartialC<Props>], any, any, unknown>, key: string, value: TypeOfPartialProps<Props>): void', gave the following error.
    Argument of type 'PartialC<{ rowHeight: ValueOfC<{ readonly EXTRA_TALL: "EXTRA_TALL"; readonly MEDIUM: "MEDIUM"; readonly SHORT: "SHORT"; readonly TALL: "TALL"; }>; tableViewMode: UnionC<[LiteralC<"scroll">, LiteralC<...>]>; }>' is not assignable to parameter of type 'IntersectionType<[TypeC<Props>, PartialC<Props>], any, any, unknown>'.
      Property 'types' is missing in type 'PartialC<{ rowHeight: ValueOfC<{ readonly EXTRA_TALL: "EXTRA_TALL"; readonly MEDIUM: "MEDIUM"; readonly SHORT: "SHORT"; readonly TALL: "TALL"; }>; tableViewMode: UnionC<[LiteralC<"scroll">, LiteralC<...>]>; }>' but required in type 'IntersectionType<[TypeC<Props>, PartialC<Props>], any, any, unknown>'.ts(2769)
index.d.ts(781, 12): 'types' is declared here.
userSettings.tsx(146, 10): The call would have succeeded against this implementation, but implementation signatures of overloads are not externally visible.
```

Reply:
this is where you intersect the type with an empty object like in https://github.com/determined-ai/determined/blob/874fa07239a008017163090f835456c8fdea9ee3/webui/react/src/pages/F_ExpList/F_ExperimentList.settings.ts#L26-L41